docs/src/cli/install.md

```diff
@@ -198,7 +198,7 @@ Install Homebrew (if not already installed), check "Install Homebrew" at
 Install the necessary tools and libraries using Homebrew:
 
 ```bash
-brew install rust pkg-config libudev protobuf llvm coreutils
+brew install rust pkg-config libudev protobuf llvm coreutils rustup
```

Comment:
why do we want it from `brew` instead of the install instructions on rustup.rs, like on ln166? i think just move the "install rust" section above the platform sections

aside: separate change, but we should probably only be linking to rustup.rs and brew.sh, rather than copypastaing their commands here. should they change, we'll be out of sync

Reply:
> aside: separate change, but we should probably only be linking to rustup.rs and brew.sh, rather than copypastaing their commands here. should they change, we'll be out of sync

Found some duplicate stuff elsewhere and agree with the sentiment so took this up: https://github.com/anza-xyz/agave/pull/4908

Reply:
I added the rustup installer via brew there because some users installing `rust` via brew seem to expect having rustup installed by it. which it does not. then if they attempt to install the solana cli,  part of it will silent fail for app devs. specifically when platform tools attempts to use the `rustup` command to set the solana toolchain